torch/onnx/symbolic_opset9.py

```diff
@@ -1950,15 +1944,47 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
 
+
 def baddbmm(g, self, batch1, batch2, beta, alpha):  
     dtype = self.type().scalarType()    
     batch_mul = matmul(g, batch1, batch2)       
     mul_a = mul(g, batch_mul, g.op("Cast", alpha, to_i=sym_help.cast_pytorch_to_onnx[dtype]))       
     mul_b = mul(g, self, g.op("Cast", beta, to_i=sym_help.cast_pytorch_to_onnx[dtype]))
     return add(g, mul_a, mul_b)
 
+
 def gelu(g, self):
     _sqrt2 = 1.4142135623730951
     erf = g.op('Erf', div(g, self, torch.tensor(_sqrt2)))
     erf_plusone = add(g, erf, g.op('Constant', value_t=torch.tensor(1, dtype=torch.float)))
     return mul(g, mul(g, self, erf_plusone), g.op('Constant', value_t=torch.tensor(0.5, dtype=torch.float)))
+
+
+@parse_args('v', 'i', 'v', 'v', 'f', 'i')
+def group_norm(g, input, num_groups, weight, bias, eps, cudnn_enabled):
+    input_sizes = input.type().sizes()
+    shape = [1, input_sizes[0] * num_groups, input_sizes[1] / num_groups, -1]
```

Comment:
In fact it might be possible to remove the dependency on explicit `input_sizes`. 

- For the first reshape, reshape to [0, num_groups, -1], the effect will be [batch, channel, h, w] => [batch, num_groups, channel / num_groups * h * w].
- For `weight` and `bias`, consider the below comment regarding broadcasting.
- For the second reshape, just reshape back to the original input shape: `g.op('Reshape', norm_reshaped, g.op("Shape", input))`.
- For the last `weight` and `bias`, it seems we still need the rank information of input. We can remove this dependency as well if we add two more reshapes to the norm tensor, but that might not worth the effort as rank info is usually static. 

The motivation is that we won't be able to support dynamic input sizes if we depend on static `input_sizes`.

Reply:
Thanks @BowenBao, apparently broadcasting is not supported for InstanceNormalization.
So I don't think we can avoid accessing the size for weight_ and bias_

Reply:
Ok, but I think we can still construct weight_ and bias_ with just num_groups, if in the first reshape we keep batch and channel axes separated.

Reply:
True, I gonna update the first two parts anyways. But I agree that it might not be worth it to add two reshapes since rank here could be N (it's not limited in spec). I'll leave a note in case anyone needed this op with dynamic rank in future.